### ISSUE TITLE
Follow Web Content Accessibility Guidelines (WCAG) on the webapp

### ISSUE THREAD

**Issue by liviopv** [Automattic/simplenote-electron#1935]
Reproducing feature suggestion from a visually-impaired user, I'm not entirely sure if it applies.

Translated from Portuguese:

> In the web app, it's already possible to use a screen reading software (used by visually-impaired people) using keyboard commands. However, it's necessary to implement the guidelines from WCAG to improve your user experience, promoting accessibility. To give examples, 1) when going through the editing fields with the Tab key and screen reading software on, the names of fields are not announced; 2) when you open note to edit, the screen reading software doesn't announce that new content was loaded on the page.

### What
Make the app fully compliant to https://www.w3.org/WAI/standards-guidelines/wcag/

### Why
Because it aligns with Automattic's core values and makes Simplenote accessible to a new set of users.

User-report: #2106972-zen

**Comment by dcalhoun** [comment]
@belcherj I believe achieving WCAG compliance could be a larger project that is likely worth splitting up into multiple tickets. We would likely also need to decide which version of WCAG to target (e.g. WCAG 2.1) and what level of compliance (e.g. AA) we would like to achieve. In the meantime, I wanted to capture some resources I am familiar with that could help us approach the larger project.

[Deque](https://www.deque.com) provides a variety of services that can help access and improve accessibility of products.

* [axe DevTools](https://www.deque.com/axe/browser-extensions/): Free Chrome extension that can automatically detect certain accessibility issues providing tooling and information on how to address them. The paid version allows integrating the automated tooling into CI suites.
* [Deque University](https://www.deque.com/training/online-deque-university/): Online training resources for self-paced accessibility learning. This can be helpful for understanding the nuances of WCAG and how to apply them to a product.
* [Audits & Training Sessions](https://www.deque.com/services/): Deque also provides audits and training sessions to help teams assess their status and equip themselves for building accessible products.
* [axe Auditor](https://www.deque.com/axe/auditor/): Tool to help define and run manual accessibility test cases.

There are obviously other companies that provide similar tooling and services, but Deque just happens to be the one with which I am most familiar.

From my perspective, a good first step could be leveraging the axe DevTools to address the issues that it finds.

<img width="1540" alt="axe-devtools-example" src="https://user-images.githubusercontent.com/438664/110178284-d62c6980-7dcb-11eb-9602-b139232e454c.png">

Hope this information is helpful moving forward!
